src/repo/zcl_abapgit_dot_abapgit.clas.abap

```diff
@@ -72,43 +76,42 @@ CLASS zcl_abapgit_dot_abapgit DEFINITION
         zcx_abapgit_exception .
     METHODS get_requirements
       RETURNING
-        VALUE(rt_requirements) TYPE zif_abapgit_dot_abapgit=>ty_requirement_tt.
+        VALUE(rt_requirements) TYPE zif_abapgit_dot_abapgit=>ty_requirement_tt .
     METHODS set_requirements
       IMPORTING
-        it_requirements TYPE zif_abapgit_dot_abapgit=>ty_requirement_tt.
-
+        !it_requirements TYPE zif_abapgit_dot_abapgit=>ty_requirement_tt .
     METHODS get_i18n_langs
```

Comment:
I'd keep just one interface (I mean string vs table)

Reply:
I didn't want to touch the settings pages, too. let's do it in separate PR.